create-minikube-pvc.sh

```diff
@@ -9,15 +9,14 @@ apiVersion: "v1"
 kind: "PersistentVolume"
 metadata:
   name: "$PV_NAME"
+  labels:
+    kubevirt.io/os: "$PV_NAME"
 spec:
   capacity:
     storage: "$PV_SIZE"
   accessModes:
     - "ReadWriteOnce"
   persistentVolumeReclaimPolicy: Delete
-  claimRef:
-    namespace: default
-    name: "$PV_NAME"
```

Comment:
why this is going away?

Reply:
because this is not working in minishift (i didn't have time to scan why). I replaced it with label selector, which is working for sure.

Reply:
OK, tnx